### PR TITLE
Updated links in dev-setup.md and user-guide.md.

### PR DESCRIPTION
**Pull request by sharajat**
Updated hyper-links for docs -:
1. dev-setup.md
2. user-guide.md

@maelk @digambar15  Kindly Review.

### CODE REVIEW

**Review by maelk — APPROVED** [review]

File: docs/user-guide.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ This feature helps user to avoid runtime failures.
...
     Follow setup documentation guide for setup.
-[Setup documentation](docs/dev-setup.md)
+[Setup Documentation](dev-setup.md)
 
 ## HardwareClassificationController Example
 
 The following is a sample CRD of a `HardwareClassificationController` resource
-([YAML PATH](config/samples/metal3.io_v1alpha1_hardwareclassification.yaml)),
+([YAML PATH](../config/samples/metal3.io_v1alpha1_hardwareclassification.yaml)),
 it includes its metadata and specification section.
```
Comment (digambar15) [inline docs/user-guide.md]: remove "its"

File: docs/dev-setup.md [hunk 2]
```diff
@@ -56,5 +56,5 @@ Please follow metal3 dev guide for setting up above prerequisites -
    make run
    ```
 
-Note: Setup is completed here. To use HWCC follow HWCC user-guide
-(Link will be shared soon)
+Note: Setup is completed here. To use HWCC follow HWCC
```
Comment (digambar15) [inline docs/dev-setup.md]: Only keep to use HWCC, follow [User guide](user-guide.md)

Anyway you included link to guide so don't need mention specifically HWCC again.

**Review by digambar15 — CHANGES_REQUESTED** [review]
Few improvements needed

File: docs/dev-setup.md (hunk 2 above)
Comment (sharajat) [reply docs/dev-setup.md]: done.

File: docs/user-guide.md (hunk 1 above)
Comment (sharajat) [reply docs/user-guide.md]: done.

**Pull request merged**